content/docs/1.4.0/references/settings.md

```diff
@@ -286,6 +286,38 @@ Longhorn blocks support bundle creation when reaching the upper bound of the lim
 > Default: `false`
 
 The setting enables fast replica rebuilding feature. It relies on the checksums of snapshot disk files, so setting the snapshot-data-integrity to **enable** or **fast-check** is a prerequisite.
+### Snapshot
```

Comment:
Add an extra empty line above to make the doc format well.

Reply:
Fixed. Thank you.